src/components/tip/tip.tsx

```diff
@@ -18,7 +18,7 @@ export interface TipProps {
   emoji?: string;
 }
 
-const Tip: React.FC<TipProps> & {
+export const Tip: React.FC<TipProps> & {
```

Comment:
It's already exported as default below

Reply:
Moved all the components to default exports and removed named exports